### PR TITLE
Add HA Proxy setup and Collect HA Proxy logs

### PR DESCRIPTION
**Pull request by Men-hau**

### CODE REVIEW

**Review by Men-hau — COMMENTED** [review]
HaProxy configuration Update

File: docs/xdr/features/collect/integrations/application/haproxy.md [hunk 1]
```diff
@@ -9,8 +9,40 @@ HAProxy is a free, open source software that provides a high availability load b
 
 {!_shared_content/operations_center/integrations/generated/haproxy_do_not_edit_manually.md!}
 
-## Configure
-As of now, the main solution to collect HAProxy logs leverages the Rsyslog recipe. Please share your experiences with other recipes by editing this documentation.
+## HaProxy Configuration
```
Comment (otetard) [inline docs/xdr/features/collect/integrations/application/haproxy.md:12]:
```suggestion
## HAProxy Configuration
```

File: docs/xdr/features/collect/integrations/application/haproxy.md [hunk 2]
```diff
@@ -9,8 +9,40 @@ HAProxy is a free, open source software that provides a high availability load b
...
+
+1- Setup HaProxy following the official documentation on section below
```
Comment (otetard) [inline docs/xdr/features/collect/integrations/application/haproxy.md:14]:
```suggestion
1- Setup HAProxy following the official documentation on section below
```

File: docs/xdr/features/collect/integrations/application/haproxy.md [hunk 3]
```diff
@@ -9,8 +9,40 @@ HAProxy is a free, open source software that provides a high availability load b
...
+HaProxy configuration default file `haproxy.cfg` will be found on repo `/etc/haproxy`
```
Comment (otetard) [inline docs/xdr/features/collect/integrations/application/haproxy.md:15]:
```suggestion
HAProxy configuration default file `haproxy.cfg` will be found in the directory `/etc/haproxy`
```

File: docs/xdr/features/collect/integrations/application/haproxy.md [hunk 4]
```diff
@@ -9,8 +9,40 @@ HAProxy is a free, open source software that provides a high availability load b
...
+
+2- Add the following line to this file as the 1st line of the `global` section 
+````
+        log 127.0.0.1    local2
+````
+
+3- Restart HaProxy for configuration to be taken account and verify the status
```
Comment (otetard) [inline docs/xdr/features/collect/integrations/application/haproxy.md:22]:
```suggestion
3- Restart HAProxy for configuration to be taken account and verify the status
```

File: docs/xdr/features/collect/integrations/application/haproxy.md [hunk 5]
```diff
@@ -9,8 +9,40 @@ HAProxy is a free, open source software that provides a high availability load b
...
+````
+	sudo systemctl restart HaProxy.conf && systemctl status HaProxy.conf
```
Comment (otetard) [inline docs/xdr/features/collect/integrations/application/haproxy.md:24]:
```suggestion
	sudo systemctl restart haproxy && systemctl status haproxy
```

File: docs/xdr/features/collect/integrations/application/haproxy.md [hunk 6]
```diff
@@ -9,8 +9,40 @@ HAProxy is a free, open source software that provides a high availability load b
...
+````
+
+HaProxy logs are now generated on your machine.
```
Comment (otetard) [inline docs/xdr/features/collect/integrations/application/haproxy.md:27]:
```suggestion
HAProxy logs are now generated on your machine.
```

File: docs/xdr/features/collect/integrations/application/haproxy.md [hunk 7]
```diff
@@ -9,8 +9,40 @@ HAProxy is a free, open source software that provides a high availability load b
...
+
+After HaProxy has been setup and configured, the logs have to be sent to a Rsyslog collector then forward to SEKOIA.IO
```
Comment (otetard) [inline docs/xdr/features/collect/integrations/application/haproxy.md:29]:
```suggestion
After HAProxy has been setup and configured, the logs have to be sent to a Rsyslog collector then forward to SEKOIA.IO
```

File: docs/xdr/features/collect/integrations/application/haproxy.md [hunk 8]
```diff
@@ -9,8 +9,40 @@ HAProxy is a free, open source software that provides a high availability load b
...
+
+Here is an example of HaProxy configuration file on Rsyslog to be used (to be adapted):
```
Comment (otetard) [inline docs/xdr/features/collect/integrations/application/haproxy.md:31]:
```suggestion
Here is an example of HAProxy configuration file on Rsyslog to be used (to be adapted):
```

**Review by CharlesLR-sekoia — APPROVED** [review]

**Pull request merged**